apps/omg_api/test/api/core_test.exs

```diff
@@ -30,28 +30,27 @@ defmodule OMG.API.CoreTest do
     alice: alice,
     bob: bob
   } do
-    parametrized_tester = fn {input1, input2, spender1, spender2} ->
+    parametrized_tester = fn inputs ->
       raw_tx =
-        Transaction.new(
-          [input1, input2] |> Enum.map(fn {blknum, txindex, oindex, _} -> {blknum, txindex, oindex} end),
-          eth(),
-          [{alice, 7}, {bob, 3}] |> Enum.map(fn {newowner, amount} -> {newowner.addr, amount} end)
-        )
+        inputs
+        |> Enum.map(fn {blknum, txindex, oindex, _} -> {blknum, txindex, oindex} end)
+        |> Transaction.new([{alice.addr, eth(), 7}, {bob.addr, eth(), 3}])
 
-      encoded_signed_tx = TestHelper.create_encoded([input1, input2], eth(), [{alice, 7}, {bob, 3}])
+      encoded_signed_tx = TestHelper.create_encoded(inputs, [{alice, eth(), 7}, {bob, eth(), 3}])
```

Comment:
what about https://github.com/omisego/elixir-omg/pull/278#discussion_r236172001 ? that was the main issue previously - we want to not have all this diff in tests.

Reply:
Sorry, I applied it only to spenders.